src/icons/collections/IconCollections.js

```diff
@@ -24,7 +24,7 @@ const IconCollections = ({
     width = 32,
 }: Props) => (
     <AccessibleSVG
-        className={classNames('icon-collections', className, {
+        className={classNames('IconCollections', className, {
```

Comment:
Common component class names should be prefixed with `bdl-`, so this would be `bdl-IconCollections`.

Reply:
Ah I was under the impression that only shared variable names and common class names  such as "Button" were prefixed with `bdl`. Will fix.